L1 phase 2: produce CaloJet/Tau

cecilecaillol opened the pull request:
#### PR description:

Updated Phase2 L1 CaloJet/Tau code from CMSSW_10_5_0_pre1 release to new data formats in more recent release and added minimum code necessary to produce CaloJets/Taus (L1Trigger/L1CaloTrigger/).

#### PR validation:

Ran python test script to test full CaloJet sequence (L1Trigger/L1CaloTrigger/python/L1CaloJets_cff.py) using latest VBFHiggsTauTau samples. Also successfully produced efficiency plots for CaloTaus.

Rebasing from https://github.com/cms-l1t-offline/cmssw/pull/861

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Rebasing from https://github.com/cms-l1t-offline/cmssw/pull/861

sudhaahuja committed:
seeding algorithm updates for phi smoothing

minor change

emacdonald16 committed:
Track-objects in CMSSW11; included prompt vs displaced track flags

(cherry picked from commit dad89184df2d1a46a79512186afa8bf27f27183e)

emacdonald16 committed:
name changes for CMS guidelines

(cherry picked from commit 3eb74bb80c19fd2d510f5e41248f6f16256c4103)

emacdonald16 committed:
minor name change

(cherry picked from commit 78122e62614ca37f2a106da971e29e9c66fcbdc3)

Commit:
first version of working emulator with HLS

(cherry picked from commit 4e4619fb35ba5e1013f1d218d5db21bccd110321)

Commit:
first complete version of the cmssw emulator

(cherry picked from commit 3a9823f06b0b3b22a675106ba793629d1cff74b6)

Commit:
combine met emulator variants into a single plugin

(cherry picked from commit 6d654237d3a6967b3fb70ab626fb6d7be72950fb)

Commit:
make table sizes configurable

(cherry picked from commit 9d45304d9223572d0f178d7b97fbf323869a4997)

Commit:
add emulator with new types

(cherry picked from commit 95c3e2894faf421d98ba58eabed9595cdb9b5a02)

Commit:
cleanup comments

(cherry picked from commit f9e0c919fe3285ed799c506b41a95a77dc46ac7c)

Commit:
remove old version

(cherry picked from commit ab239bfa9824678e9a4a13975bf0dfe472d6ced5)

mtosi committed:
cleaning MTV

clean

Update MTVHistoProducerAlgoForTracker.cc

Update MTVHistoProducerAlgoForTracker.cc

fix code-format

fabiocos committed:
Rename MTDTopologyBuilder as MTDPixelTopologyBuilder

fabiocos committed:
Propagate ETL v5 start copy and offset into PMTDParameters

fabiocos committed:
Add ETL structure information to MTDTopology

fabiocos committed:
Redefine face layour using array, development commit

fabiocos committed:
horizontal and vertical shift functions

makortel committed:
Allow edmodule.clone(pset) for pset to override parameters

fabiocos committed:
Update topology, use for MTDDetSector navigation, ok for hshift

fabiocos committed:
Fixes to the vertical navigation

fabiocos committed:
Turn test of REcoMTD/DetLayers into a unit test, reference to be provided in Geometry/TestReference

fabiocos committed:
Remove useless argument in compatibleDetsLine

fabiocos committed:
Update for missing vshift, provide and use ifno about closest module in row if present

fabiocos committed:
Clean MEssageLogger in unit tests

fabiocos committed:
Silence debugging printout

felicepantaleo committed:
Use caloparticles when boundary crossed, otherwise use simclusters.

fabiocos committed:
Protect MTDTopology construction for legacy scenarios with 1 ETL disc

fabiocos committed:
scram b code-format

cmsbuild commented:
-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22446

- This PR adds an extra 56KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

Code check has found code style and quality issues which could be resolved by applying following patch(s)

- **code-checks**:
https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22446/code-checks.patch
e.g. `curl https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22446/code-checks.patch | patch -p1`
You can also run `scram build code-checks` to apply code checks directly

- **code-format**:
https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22446/code-format.patch
e.g. `curl https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22446/code-format.patch | patch -p1`
You can also run `scram build code-format` to apply code format directly

cmsbuild committed:
Merge pull request #33597 from davidlange6/dl210501

Validation/Geometry apply futurize stage1; use six.iteritems; use python3 for matplotlib

perrotta committed:
Fix typo in the fillDescriptions

Commit:
Merged l1t-Victor_Nov_120X from repository cecilecaillol with cms-merge-topic

Commit:
scram code format

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22464

- This PR adds an extra 68KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

cmsbuild commented:
A new Pull Request was created by  @cecilecaillol  for master.

It involves the following packages:

DataFormats/L1TCalorimeterPhase2
L1Trigger/L1CaloTrigger
L1Trigger/TrackFindingTracklet


@cmsbuild, @rekovic, @srimanob, @cecilecaillol, @kpedro88 can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @erikbutz, @skinnari, @rovere this is something you requested to watch as well.
@silviodonato, @dpiparo, @qliphy you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

cecilecaillol commented:
please test

Commit:
Merged l1t-AuantizedPuppiMet_120X from repository cecilecaillol with cms-merge-topic

Commit:
scram format

cmsbuild committed:
Merge pull request #33614 from bsunanda/Run4-hgx285

Run4-hgx285 Update some of the remaining validation classes in Validation/HGCalValidation

cmsbuild commented:
-1

**Failed Tests**: RelVals RelVals-INPUT 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/summary.html
**COMMIT**: 954ad0971998caf01a03d4d0e1b73fc6c82d1801
**CMSSW**: CMSSW_12_0_X_2021-05-03-2300/slc7_amd64_gcc900
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/33612/14831/install.sh` to create a dev area with all the needed externals and cmssw changes.

**CMS Clang-Tidy warnings**: There are  Clang-Tidy warnings. See https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/llvm-analysis/cmsclangtidy.txt for details.

## RelVals

- [23234.0](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrix-results/23234.0_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal)
```
----- Begin Fatal Exception 04-May-2021 12:22:41 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'FEVTDEBUGHLToutput_step'
   [2] Prefetching for module PoolOutputModule/'FEVTDEBUGHLToutput'
   [3] Prefetching for module L1TPFProducer/'l1pfProducerBarrel'
   [4] Calling method for module L1TPFCaloProducer/'pfClustersFromCombinedCaloHCal'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: std::vector<l1tp2::CaloTower>
Looking for module label: L1EGammaClusterEmuProducer
Looking for productInstanceName: 

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```
- [34634.0](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrix-results/34634.0_TTbar_14TeV+2026D76+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal)
```
----- Begin Fatal Exception 04-May-2021 12:25:00 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'FEVTDEBUGHLToutput_step'
   [2] Prefetching for module PoolOutputModule/'FEVTDEBUGHLToutput'
   [3] Prefetching for module L1TPFProducer/'l1pfProducerBarrel'
   [4] Calling method for module L1TPFCaloProducer/'pfClustersFromCombinedCaloHCal'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: std::vector<l1tp2::CaloTower>
Looking for module label: L1EGammaClusterEmuProducer
Looking for productInstanceName: 

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```
- [28234.0](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrix-results/28234.0_TTbar_14TeV+2026D60+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal)
```
----- Begin Fatal Exception 04-May-2021 12:25:05 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'FEVTDEBUGHLToutput_step'
   [2] Prefetching for module PoolOutputModule/'FEVTDEBUGHLToutput'
   [3] Prefetching for module L1TPFProducer/'l1pfProducerBarrel'
   [4] Calling method for module L1TPFCaloProducer/'pfClustersFromCombinedCaloHCal'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: std::vector<l1tp2::CaloTower>
Looking for module label: L1EGammaClusterEmuProducer
Looking for productInstanceName: 

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```
<details>
<summary>Expand to see more relval errors ...</summary>

- [34834.999](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrix-results/34834.999_TTbar_14TeV+2026D76PU_PMXS1S2PR+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+PREMIX_PremixHLBeamSpot14PU+DigiTriggerPU+RecoGlobalPU+HARVESTGlobalPU)
</details>



## RelVals-INPUT

- [23234.0](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/23234.0_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal)```23234.0_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal/step2_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal.log```
- [23234.21](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/23234.21_TTbar_14TeV+2026D49_ProdLike+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+MiniAOD)```23234.21_TTbar_14TeV+2026D49_ProdLike+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+MiniAOD/step2_TTbar_14TeV+2026D49_ProdLike+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+MiniAOD.log```
- [23234.103](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/23234.103_TTbar_14TeV+2026D49Aging3000+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal)```23234.103_TTbar_14TeV+2026D49Aging3000+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal/step2_TTbar_14TeV+2026D49Aging3000+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal.log```
<details>
<summary>Expand to see more relval errors ...</summary>

- [23234.9](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/23234.9_TTbar_14TeV+2026D49_vectorHits+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal)
- [34634.0](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/34634.0_TTbar_14TeV+2026D76+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTrigger+RecoGlobal+HARVESTGlobal)
- [23434.21](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14831/runTheMatrixINPUT-results/23434.21_TTbar_14TeV+2026D49PU_ProdLike+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14INPUT+DigiTriggerPU+RecoGlobalPU+MiniAODPU)
</details>

Commit:
restore tau DQM for previous eras

Commit:
fix syntax of removal of idDecayModeOldDMs in DQM

skinnari commented on L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py:
```diff
@@ -38,4 +38,3 @@
                                                TrackQuality =cms.bool(False),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
-
```
it seems like it is a nonsense change in the L1Trigger/TrackFindingTracklet package. Can it please be removed to avoid confusion about PRs that made changes to this package? thanks.

cecilecaillol replied:
Fixed

cmsbuild committed:
Merge pull request #33523 from pieterdavid/stripG2PCL_removeShallow

Merge ShallowGainCalibration logic in SiStripGainsPCLWorker and a minor bugfix

pieterdavid committed:
Use esConsumes for SiStripFedZeroSuppression

mtosi committed:
fix bug

cmsbuild committed:
Merge pull request #33595 from bsunanda/Run4-hgx282Z

Run4-hgx282Z Remove overlaps in the V15 geometry of HGCal

pieterdavid committed:
esConsumes for SiStripClusterToDigiProducer

pieterdavid committed:
esConsumes for SiStripHitEffFromCalibTree

Commit:
Merged l1t-Victor_Nov_120X from repository cecilecaillol with cms-merge-topic

Commit:
minor fix, add empty line

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22473

- This PR adds an extra 68KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

cmsbuild commented:
Pull request #33612 was updated. @cmsbuild, @rekovic, @srimanob, @cecilecaillol, @kpedro88 can you please check and sign again.

mtosi committed:
fix code-format

Commit:
import missing era modifiers to DQM config

cmsbuild committed:
Merge pull request #33474 from gartung/gartung-tbb2021

Changes required to compile with TBB 2021.2

Commit:
Cleanup of data/simulation analyzer modules

Commit:
Code format

Commit:
Move to test

justinasr committed:
Allow to suppress ExternalLHEProducer exception if not all events were processed

justinasr committed:
Merge branch 'master' into from-CMSSW_11_0_0_pre7-externallheproducer

mmusich committed:
fix missing leftover Timing parameter in fed_dqm_sourceclient-live_cfg.py

czangela committed:
remove remaining Timing parameters

czangela committed:
remove more Timing

Commit:
Clean up a few classes for the use of ESGetToken and add a script for plots

Commit:
Code Check

justinasr committed:
Fix incorrect variable name

cmsbuild committed:
Merge pull request #33618 from perrotta/fixTypoInHGCRecHitNavigator

Fix typo in the fillDescriptions method of RecoParticleFlow/PFClusterProducer/interface/HGCRecHitNavigator.h

qliphy committed:
fix 33544

cmsbuild committed:
Merge pull request #33625 from mmusich/fixMissingTiming

fix missing leftover Timing parameter in fed_dqm_sourceclient-live_cfg.py

Commit:
Initialize id variable to silence warning

Commit:
Merged l1t-Victor_Nov_120X from repository cecilecaillol with cms-merge-topic

Commit:
fix calo tower input collection

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22492

- This PR adds an extra 16KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

cmsbuild commented:
Pull request #33612 was updated. @cmsbuild, @rekovic, @srimanob, @cecilecaillol, @kpedro88 can you please check and sign again.

cecilecaillol commented:
please test

justinasr committed:
Applied code checks patch

cmsbuild committed:
Merge pull request #33517 from jainshilpi/HtoZZULID

Adding HtoZZ UL ID configs

cmsbuild committed:
Merge pull request #33628 from qliphy/fixh7cocurrent

Set generateConcurrently to False to fix Herwig workflow issue

civanch committed:
updated physics configurations

cmsbuild committed:
Merge pull request #33488 from lecriste/MultiCluster_to_Trackster_120

[HGCAL] Deprecate MultiCluster

cmsbuild committed:
Merge pull request #33627 from bsunanda/Phase2-hgx285X

Phase2-hgx285X Clean up a few classes for the use of ESGetToken and add a script for plots

civanch committed:
code format

cmsbuild commented:
-1

**Failed Tests**: RelVals-INPUT 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14873/summary.html
**COMMIT**: cf9d35f705bc079440b4deea49015dae0501eddd
**CMSSW**: CMSSW_12_0_X_2021-05-04-2300/slc7_amd64_gcc900
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/33612/14873/install.sh` to create a dev area with all the needed externals and cmssw changes.

**CMS Clang-Tidy warnings**: There are  Clang-Tidy warnings. See https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14873/llvm-analysis/cmsclangtidy.txt for details.

## RelVals-INPUT



## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - ROOTFileChecks: Some differences in event products or their sizes found
 - Reco comparison results: 4 differences found in the comparisons
 - DQMHistoTests: Total files compared: 37
 - DQMHistoTests: Total histograms compared: 2662646
 - DQMHistoTests: Total failures: 7
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 2662617
 - DQMHistoTests: Total skipped: 22
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 36 files compared)
 - Checked 155 log files, 37 edm output root files, 37 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_0_X_2021-05-04-2300+44b834/42624/triggerResults): no differences found

pieterdavid committed:
Fix compilation warning

cmsbuild committed:
Merge pull request #33573 from b-fontana/ecal_condobject_container_iterator_fix

EcalCondObject container iteration fix

cmsbuild committed:
Merge pull request #33582 from dildick/from-CMSSW_12_0_X_2021-04-19-2300-CSCValDQMReport

Cleanup of data/simulation analyzer modules in CSCTriggerPrimitives

cmsbuild committed:
Merge pull request #33631 from mrodozov/fix-warning-eventFilterUtils

Initialize id variable to silence warning

missirol committed:
avoiding trailing whitespaces in config dump

cmsbuild committed:
Merge pull request #33624 from justinasr/from-CMSSW_11_0_0_pre7-externallheproducer

Allow to suppress ExternalLHEProducer exception if not all events were processed

Commit:
fixing ESHandle

Commit:
Addition to PixelDQM: cluster charge corrected with template and also the template correction itself

cmsbuild commented:
-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22499

- This PR adds an extra 28KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

Code check has found code style and quality issues which could be resolved by applying following patch(s)

- **code-format**:
https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22499/code-format.patch
e.g. `curl https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22499/code-format.patch | patch -p1`
You can also run `scram build code-format` to apply code format directly

Commit:
code format

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22502

- This PR adds an extra 72KB to repository

- Found files with invalid states:
  - L1Trigger/TrackFindingTracklet/python/L1TrackletTracks_cff.py:
    - Added: 2b5f12a0ab60bd253dd14ff576cb5e2b10e92643
    - Deleted: 5e7f14cdde851633c18789cd2a054de0a0fa2470

cmsbuild commented:
Pull request #33612 was updated. @cmsbuild, @rekovic, @srimanob, @cecilecaillol, @kpedro88 can you please check and sign again.

cecilecaillol commented:
please test

Commit:
code-format patch

Commit:
Move DBTemplate access to token

Commit:
code patch (again)

felicepantaleo committed:
modify Tracksters to have a float vertex multiplicity

felicepantaleo committed:
using float multiplicity for fraction

mseidel42 committed:
Allow for increased mass precision in NanoGen

Commit:
Merged l1t-AuantizedPuppiMet_120X from repository cecilecaillol with cms-merge-topic

Commit:
address Silvio's comments

Commit:
code format

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44b834/14885/summary.html
**COMMIT**: 9f7cbdcf43c7c9c777a0d5970d60b667f3876fce
**CMSSW**: CMSSW_12_0_X_2021-05-04-2300/slc7_amd64_gcc900
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/33612/14885/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - ROOTFileChecks: Some differences in event products or their sizes found
 - Reco comparison results: 4 differences found in the comparisons
 - DQMHistoTests: Total files compared: 37
 - DQMHistoTests: Total histograms compared: 2662646
 - DQMHistoTests: Total failures: 7
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 2662617
 - DQMHistoTests: Total skipped: 22
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 36 files compared)
 - Checked 155 log files, 37 edm output root files, 37 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_0_X_2021-05-04-2300+44b834/42632/triggerResults): no differences found

cecilecaillol commented:
+l1

felicepantaleo committed:
add premix modifier

cmsbuild committed:
Merge pull request #33634 from missirol/devel_configTrailingWhitespaces

avoiding trailing whitespaces in config dump

hyunyong committed:
add default dPhi value

felicepantaleo committed:
add fraction cut

felicepantaleo committed:
set fraction cut to 0.05

Commit:
Fix the SiPixelTemplateDBObject Token with the beginRun transition

Commit:
Fix the SiPixelTemplateDBObject Token with the beginRun transition

fabiocos committed:
Adapt gap parameters for old ETL scenarios, where it was not used (D49 is based on I10, thanks to Pablo)

parbol committed:
MTDparameters.xml updated for compatibility with old geometries

Commit:
code-format patch

jeongeun committed:
change default cfi name produceMuons to muonSeedGenerator

civanch committed:
clean-up user actions

felicepantaleo committed:
disable fraction cut

davidlange6 committed:
move unit tests to python3

parbol committed:
Update mtdParameters for D49 and D60 (Bug fix)

mmusich committed:
TrackDistanceValueMapProducer: check the size of the reference vector before copying

Commit:
Reduce the number of plots generated with tempalte info, move the TrackerGeometry and Topology loading at BeginRun

sroychow committed:
remove Fall17V1 variables from electron table

sroychow committed:
update dqm according to fall17v1 removal

fabiocos committed:
Update mtd validation configuration to scenario D76, use up-to-date fragments and options

Commit:
Some code semplification applied

Commit:
Merged l1t-Emily_TkObjects_120X from repository cecilecaillol with cms-merge-topic

Commit:
code format

Commit:
move nuplizer files to correct directory

Commit:
remove unused variable

Commit:
Merged l1t-Emily_TkObjects_120X from repository cecilecaillol with cms-merge-topic

lecriste committed:
Avoid duplicate plots with 'all' option

Commit:
reorganize files

swagata87 committed:
move EgammaLocalCovParamDefaults.h from RecoEgamma to RecoEcal package

cmsbuild committed:
Merge pull request #33528 from cms-tau-pog/CMSSW_11_3_X_tau-pog_boostedTaus

Add boosted taus to NanoAOD

cmsbuild committed:
Merge pull request #33513 from cms-tau-pog/CMSSW_11_3_X_tau-pog_nanoAODcleanUp

Clean up tau related nanoAOD content

cmsbuild committed:
Merge pull request #33603 from ggovi/condcore-lumicondforhlt-7-120X

Bug fix for tag lock clean up at session closing time

cmsbuild committed:
Merge pull request #33649 from fabiocos/fc-mtdvalidation

MTD validation: update standalone test configuration to scenario D76, use up-to-date fragments

cmsbuild committed:
Merge pull request #33647 from parbol/MTDParametersUpdate

Mtd parameters update for D49 and D60 geometries (Bug fix)

Commit:
typo fix

lecriste committed:
Move plugins out of SimDataFormats

lecriste committed:
Uniform LCToCP naming

tlampen committed:
Beamspot and vertex smaring for 2017 pp ref (forward port of #33504)

fabiocos committed:
updates addressing comments by C. Vuosalo

fabiocos committed:
scram b code-format

Commit:
Added specific plot for Inner/outer modules, unuseful plot removed, disabled these plots on Online

cmsbuild committed:
Merge pull request #33483 from hqucms/dev/particle_net_mass_regression

Add ParticleNet mass regression

cmsbuild committed:
Merge pull request #33639 from lecriste/makeHGCalValidationPlots_simplify_master

[HGCAL] Simplify makeHGCalValidationPlots

cmsbuild committed:
Merge pull request #33617 from dildick/from-CMSSW_12_0_X_2021-05-03-1100-GEMCSC-Run3-prep-2

More prep work for the Run-3 GEM-CSC integrated local trigger

cmsbuild committed:
Merge pull request #33599 from theofil/renamingCFGs

Renaming cfgs and cffs

gennai committed:
added the possibility to lock a record

ggovi committed:
Added support of Tag access restrictions

casarsa committed:
Added optional histograms of BTL and ETL UncalibratedRecHits to monitor the BTL and ETL time resolutions and a ROOT macro to analyze them.

gennai committed:
fixed the compilation error

casarsa committed:
Applied code-format and renamed macro to macros

casarsa committed:
Removed the macro folder

casarsa committed:
Applied code-format

pieterdavid committed:
Avoid 'handle' in member names that are not ESHandles anymore

lecriste committed:
Uniform LCToSC naming

lecriste committed:
Uniform TSToSC naming

lecriste committed:
Forgot auto

lecriste committed:
code-format

lecriste committed:
format code-format

cmsbuild committed:
Merge pull request #33635 from arossi83/PixTemplateCorr

Addition to PixelDQM of Template correction to cluster charge

cmsbuild committed:
Merge pull request #33633 from civanch/geant4_physics_for_10.7

Updated physics configurations for Geant4

fabiocos committed:
move static constexpr as class member in MTDTopology

mmusich committed:
post PR #33635 merge fixes

mseidel42 committed:
Increase mass precision for SUSY particles

cmsbuild committed:
Merge pull request #33669 from mmusich/postMergeFix

Pixel DQM: post PR #33635 merge fixes

jeongeun committed:
replace EDProducer with clone from the default cipython and cleanup

cmsbuild committed:
Merge pull request #33638 from felicepantaleo/SimTrackstersFromCP_12_0_X

[HGCAL] SimTracksters from Caloparticles

cmsbuild committed:
Merge pull request #33662 from casarsa/mc-mtdValidationUpdate

MTD validation: optional histograms of BTL and ETL uncalibrated RecHits

ggovi committed:
Fix for incomplete query!

mmusich committed:
move from orbitNumber() to luminosityBlock() in the simulation case

cmsbuild committed:
Merge pull request #33551 from hyunyong/gemStripTopoPR_CMSSW_12_0_X_2021-04-27-1100

Remove hard-coded dphi value

Commit:
Subdirectories for GEMCSC LUTs

justinasr committed:
Use std::getenv in ExternalLHEProducer

Commit:
Add possibility of studying MB in all subvolumes of CMS

Commit:
Consolidate readmes

Commit:
Document slope value

Commit:
Code check

Commit:
Turn-off GEM-CSC integrated local trigger

cmsbuild committed:
Merge pull request #33646 from davidlange6/dl210506

move another set of python tool unit tests to python3

cmsbuild committed:
Merge pull request #33600 from bsunanda/Run3-gex73

Run3-gex73 Remove reference of DDD from plugins/dd4hep/DDEcalEndcapAlgo.cc

lecriste committed:
Static Check

lecriste committed:
code-reformat

cmsbuild committed:
Merge pull request #33678 from dildick/from-CMSSW_12_0_X_2021-05-10-1100-disable-GEM-CSC-integrated-local-trigger

Turn-off GEM-CSC integrated local trigger in valCscStage2Digis [12_0_X]

mmusich committed:
update documentation after commit 69c8898814ce3be38134d8a3a5bcea09951a537c

Commit:
Improve CSC TP quality control

Commit:
Use .length instead of strlen() in AMPTHadronizer

Resolves: https://github.com/cms-sw/cmssw/issues/33682
(or at least will try to)

Commit:
Update AMPTHadronizer.cc

cmsbuild committed:
Merge pull request #33674 from justinasr/from-CMSSW_12_0_X_2021-05-10-1100

Use std::getenv in ExternalLHEProducer

cmsbuild committed:
Merge pull request #33685 from mmusich/updateDocsInTrackerGeometryBuilder

 Geometry/TrackerGeometryBuilder: update documentation after #27448

cmsbuild committed:
Merge pull request #33637 from mseidel42/NanoGen_120

Allow for increased mass precision in NanoGen + increase mass precision of SUSY particles in NanoAod

cmsbuild committed:
Merge pull request #33632 from pieterdavid/sistrip_esconsumes_3

esConsumes migration for SiStripFedZeroSuppression, SiStripHitEffFromCalibTree, and SiStripClusterToDigiProducer

cmsbuild committed:
Merge pull request #33673 from makortel/clonePSet

Allow edmodule.clone(pset) for PSet to override parameters

cmsbuild committed:
Merge pull request #33655 from tlampen/beamspot-and-vtxsmearing-for-ppRefULMC

Beamspot and vertex smaring for 2017 pp ref (forward port of #33504)

cmsbuild committed:
Merge pull request #33621 from PFCal-dev/hgc-tpg-integration-210504

[HGCAL trigger] Fixing phi smoothing in cluster seeding

cmsbuild committed:
Merge pull request #33664 from lecriste/movePlugins

[HGCAL] Move plugins out of SimDataFormats

cmsbuild committed:
Merge pull request #33675 from bsunanda/Run3-gex74

Run3-gex74 Add possibility of studying MB in all subvolumes of CMS

mmusich committed:
migrate SLHCUpgradeSimulations/Geometry to esConsumes

dmeuser committed:
Added 2D maps of Impact Parameters for TkDQM

cmsbuild committed:
Merge pull request #33648 from mmusich/fixTrackDistanceValueMapProducer

TrackDistanceValueMapProducer: check the size of the reference vector before copying

cmsbuild committed:
Merge pull request #33672 from mmusich/SiStripMonitorClustermoveFromOrbitToLS

SiStripMonitorCluster: move from orbitNumber() to luminosityBlock()

cmsbuild committed:
Merge pull request #33606 from cecilecaillol/l1t-Emyr_120X

L1T phase-2: Update to histogram (Phase-1 like) jets

dmeuser committed:
Applied code format

cmsbuild committed:
Merge pull request #33428 from fwyzard/auto_gpu_workflows

Redesign all GPU workflows to detect if a GPU is present, and fall back to CPU otherwise

cmsbuild committed:
Merge pull request #33687 from mrodozov/patch-9

Use .length instead of strlen() in AMPTHadronizer

cmsbuild committed:
Merge pull request #33686 from dildick/from-CMSSW_12_0_X_2021-05-10-1100-more-lct-quality-control

Improve quality control for CSC trigger primitives

cmsbuild committed:
Merge pull request #33500 from mtosi/MTVcleaning

MTV fast cleaning

Dr15Jones committed:
Split ProductProvenanceRetriever into to classes

ProductProvenanceLookup is the API intended for accessing the information.
ProductProvenanceRetriever now just handles the retrieving of data from storage.

mmusich committed:
PrimaryVertexMonitoring: don't use HP flag for Alignment plots at HLT

cmsbuild committed:
Merge pull request #33626 from czangela/fix_sipixelrawtodigi_04_05_2021

Fix leftover Timing parameters

cmsbuild committed:
Merge pull request #33653 from swagata87/fixCircularDep_12XY

Fix circular dependency between RecoEgamma and RecoEcal

cmsbuild committed:
Merge pull request #33563 from jeongeun/updatemuon113X

Migrate module config in RecoMuon to use automatically generated cfipython default

Dr15Jones committed:
Moved ProductProvenanceRetriever to FWCore/Framework

This allows DataFormats/Provenance to no longer depend upon FWCore/Concurrency.

sharmaaash committed:
Update QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi.py

sharmaaash committed:
Update TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi.py

sroychow committed:
remove fall17v1 variable from nanoDQM

ianna committed:
as suggested by Vincenzo

jeongeun committed:
remove deprecated parameters

Commit:
Remove tests for root_numpy and root_pandas

mmusich committed:
introduce DiMuonVertexValidation in Alignment validation suite

Commit:
remove file testRootNumpy.py

cavana committed:
implements UPT invariant mass triggers in uGT emulator

cmsbuild committed:
Merge pull request #33696 from mmusich/noHPforHLTTracksPVAlignmentMonitoring

PrimaryVertexMonitoring: don't use HP flag for Alignment plots at HLT

jeongeun committed:
remove deprecated parameters

Commit:
fixes by Emily

sroychow committed:
add pre ul era modifiers correctly

cmsbuild committed:
Merge pull request #33693 from dmeuser/TkDQM_2Dmaps_ImpactParamter

2D Maps of Impact Parameters for TkDQM

cavana committed:
resolves style issues

Commit:
Fix compilation failure in DBG in RecoPixelVertexing/PixelVertexFinding

Martin-Grunewald committed:
HLT menu migration to 12_0_0_pre1

Commit:
remove useless cfi

Commit:
minor comment from Silvio

perrotta committed:
Deprecated parameter and header cleaning

sroychow committed:
fix es data access

ianna committed:
update unit test

cmsbuild committed:
Merge pull request #33651 from cecilecaillol/l1t-Emily_TkObjects_120X

L1T Phase-2: add various L1 Tk objects

cmsbuild committed:
Merge pull request #33677 from dildick/from-CMSSW_12_0_X_2021-05-10-1100-GEMCSC-LUTs

Subdirectories for GEM-CSC LUTs

cmsbuild committed:
Merge pull request #33659 from MilanoBicocca-pix/12_0_X_BeamSpot_LockRecords

 added the possibility to lock a record

cmsbuild committed:
Merge pull request #33619 from cecilecaillol/l1t-AuantizedPuppiMet_120X

L1T Phase-2: Quantized Puppi MET emulators

cmsbuild committed:
Merge pull request #33671 from jeongeun/recoecal113X

Migrate module config in RecoEcal to use default cfipython

cecilecaillol committed:
L1T: add HPS taus (#33681)

* Uploaded HPS Tau algorithm for new PR

(cherry picked from commit 8f55d733315824bd1c5d04c2888bbd322cc6a2ac)

* modified considering Andrea's comments

(cherry picked from commit a30f47c363191b2d39c0ff137b286d84d9770478)

* DataFormats/Phase2L1Taus is merged with DataFormats/L1TParticleFlow

(cherry picked from commit 21829a0a5bc3c2f84025848d37d1f570ef38625a)

* renamed HPSPFTauBuilder and HPSPFTauQualityCut as original

(cherry picked from commit 391723c56fcdb1cb18bbc8e53a07cd4346f1d576)

* code format

* code format

* fix BuildFile

* remove cout and switch to CaloJet

Co-authored-by: sandeepbhowmik1 <sandeep.bhowmik@cern.ch>
Co-authored-by: ccaillol <ccaillol@cern.ch>

cmsbuild committed:
Merge pull request #33720 from cms-tsg-storm/HLTmigrationTo1200pre1

HLT Menu Migration to 12_0_0_pre1

cmsbuild committed:
Merge pull request #33689 from mmusich/esConsumesSLHCUpgradeSimulations

migrate SLHCUpgradeSimulations/Geometry to esConsumes

Dr15Jones committed:
Removed unnecessary includes

cmsbuild committed:
Merge pull request #33722 from perrotta/removeDepecatedTimingParameter

Remove a deprecated "Timing" parameter from SiPixelDigiToRaw (and some header cleaning)

cmsbuild committed:
Merge pull request #33701 from sroychow/electronsremovefall17V1

EGamma: Remove Fall17V1 variables from electron table for v9 NanoAOD

Commit:
change the unsigned to int and keep the int as it was

silviodonato commented:
@cms-sw/upgrade-l2 do you have any comments?

cmsbuild committed:
Merge pull request #33707 from mrodozov/remove-rootnumpy-rootpandas

Remove tests for root_numpy and root_pandas

srimanob commented:
@silviodonato Are we OK with printf? Thanks

silviodonato commented:
instead of having 
```
if (debug){
printf(...)
}      
```
it would be much better to use directly `LogDebug(...)`
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger#LogDebug

cmsbuild committed:
Merge pull request #33708 from cavana/upt-invariant-mass_12_0_X

implements UPT invariant mass triggers in uGT emulator

cmsbuild committed:
Merge pull request #33700 from sharmaaash/master

Updated import TUNECP5

ggovi committed:
Added support of "frequent" flag/ stored timestamps moved to UTC

fabiocos committed:
Remove forgotten old commented statements (comment by P. Srimanobhas)

cmsbuild committed:
Merge pull request #33721 from sroychow/fixesaccessnano

NanoAOD:migrate modules to ESConsumes

cmsbuild committed:
Merge pull request #33660 from ggovi/condcore-conddb-tools-tag-authorization-1-120X

Setting of Tag access restrictions in command line tools

cmsbuild committed:
Merge pull request #33709 from mmusich/DiMuonVertexValidation_in_CMSSW

introduce DiMuonVertexValidation in Alignment validation suite

cmsbuild committed:
Merge pull request #33699 from Dr15Jones/ProductProvenanceRetriever

Moved ProductProvenanceRetriever to FWCore/Framework

cmsbuild committed:
Merge pull request #33705 from ianna/math-units-return-double

user-defined floating-point literals return double instead of long double

cmsbuild committed:
Merge pull request #33733 from ggovi/condcore-utilities-o2o-8-120X

Improvements of O2O steering tools

cmsbuild committed:
Merge pull request #33718 from mrodozov/RecoPixelVertexing-PixelVertexFinding-comp-failure

Fix compilation failure in DBG in RecoPixelVertexing/PixelVertexFinding

cmsbuild committed:
Merge pull request #33598 from fabiocos/fc-etllayout

MTD geometry and reconstruction: store ETL layout logic into MTDTopology, use for ETL layers navigation in reconstruction

Commit:
Merged l1t-Victor_Nov_120X from repository cecilecaillol with cms-merge-topic

Commit:
switch from printf to LogDebug

cecilecaillol commented:
I updated to a newer CMSSW release and included many irrelevant PRs. I will reopen a clean PR with the Calo jets/taus.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-33612/22695

- This PR adds an extra 28KB to repository

- Found files with invalid states:
  - L1Trigger/Phase2L1ParticleFlow/python/L1METPFProducer_cfi.py:
    - Added: 5c055aad6dbd7b68e9f96afb165f0ab93768bf9a
    - Deleted: a3612b9f645f090936f5e19db877781d8ec0ec1b
  - L1Trigger/TrackFindingTracklet/test/L1TrackObjectNtupleMaker.cc:
    - Added: ba6c6b9e11ec3e8cbd5aae66daaf0a6ec9dd6f62
    - Modified: 987f7956ae6adde84a9f94d648ec4df86888bfaa
    - Deleted: 5d86491a71ca1dace4fe86de80c55ee9c0b00448
  - L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducer.cc:
    - Added: 48150578b1491af181117258c3050379ec16be80
    - Modified: eaeba68eec78b3fa2d527a37cb74cc2ebd88a86a, 30fa8fbd01dd1534ee11ee44873b805ce188bae0, 03a332b5e48f5e77c21417f368d4eba11748ab4b, 9dcba0ef87ceff4603fa9308a4d10ad14ff12165, 5ba474f597202b107c33cef238b187c5e86bb04f
    - Deleted: 5c055aad6dbd7b68e9f96afb165f0ab93768bf9a
  - L1Trigger/Phase2L1ParticleFlow/python/L1MetPfProducer_cfi.py:
    - Added: 48150578b1491af181117258c3050379ec16be80
    - Modified: eaeba68eec78b3fa2d527a37cb74cc2ebd88a86a, 30fa8fbd01dd1534ee11ee44873b805ce188bae0, 03a332b5e48f5e77c21417f368d4eba11748ab4b, b83fdbb07569a14b5baee721dffdf3e89da5b56c, 9dcba0ef87ceff4603fa9308a4d10ad14ff12165, bc6ed5f7f0a8c733e5285d43867e08b12f77ceda
    - Deleted: 5c055aad6dbd7b68e9f96afb165f0ab93768bf9a
  - Validation/MtdValidation/macro/makeTimeResPlots.C:
    - Added: defee1c12a701204ddb88b11664260f8538493d2
    - Deleted: d3b4d1251b8e4ffcdcf56c4c9b4c3d0fd1fbb676
  - L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducerHLS.cc:
    - Added: 48150578b1491af181117258c3050379ec16be80
    - Modified: eaeba68eec78b3fa2d527a37cb74cc2ebd88a86a
    - Deleted: 30fa8fbd01dd1534ee11ee44873b805ce188bae0
  - L1Trigger/TrackFindingTracklet/test/L1TrackObjectNtupleMaker_cfg.py:
    - Added: ba6c6b9e11ec3e8cbd5aae66daaf0a6ec9dd6f62
    - Deleted: 5d86491a71ca1dace4fe86de80c55ee9c0b00448
  - L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducerNewTypes.cc:
    - Added: b83fdbb07569a14b5baee721dffdf3e89da5b56c
    - Deleted: 9dcba0ef87ceff4603fa9308a4d10ad14ff12165
  - L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.h:
    - Added: ba6c6b9e11ec3e8cbd5aae66daaf0a6ec9dd6f62
    - Modified: 976a1aec7b3bc41f94366928a521703a48141238, 987f7956ae6adde84a9f94d648ec4df86888bfaa
    - Deleted: 4e61998b2188fda1b744d7a17af2bef6e82d2cd0
  - L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducerTDR.cc:
    - Added: 9dcba0ef87ceff4603fa9308a4d10ad14ff12165
    - Deleted: bc6ed5f7f0a8c733e5285d43867e08b12f77ceda

- There are other open Pull requests which might conflict with changes you have proposed:
  - File  Configuration/AlCa/python/autoCond.py  modified in PR(s): #33548
  - File  Configuration/PyReleaseValidation/python/relval_steps.py  modified in PR(s): #32957, #33555
  - File  Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py  modified in PR(s): #33548, #33532
  - File  DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py  modified in PR(s): #33643
  - File  DQM/Integration/python/clients/beamfake_dqm_sourceclient-live_cfg.py  modified in PR(s): #33643
  - File  DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py  modified in PR(s): #33643
  - File  DQM/Integration/python/clients/beamhltfake_dqm_sourceclient-live_cfg.py  modified in PR(s): #33643
  - File  DQM/L1TMonitor/python/L1TStage2Emulator_cff.py  modified in PR(s): #33694
  - File  DataFormats/HGCalReco/interface/Trackster.h  modified in PR(s): #33548
  - File  DataFormats/L1TParticleFlow/src/classes_def.xml  modified in PR(s): #33605
  - File  DataFormats/L1Trigger/src/classes_def.xml  modified in PR(s): #33609
  - File  GeneratorInterface/LHEInterface/plugins/ExternalLHEProducer.cc  modified in PR(s): #32167, #33615
  - File  L1Trigger/CSCTriggerPrimitives/src/CSCMotherboard.cc  modified in PR(s): #33727
  - File  L1Trigger/L1TMuonEndCap/src/SectorProcessorShower.cc  modified in PR(s): #33727
  - File  L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc  modified in PR(s): #33742
  - File  PhysicsTools/NanoAOD/python/electrons_cff.py  modified in PR(s): #33666
  - File  PhysicsTools/NanoAOD/python/nano_cff.py  modified in PR(s): #32167
  - File  PhysicsTools/NanoAOD/python/nanogen_cff.py  modified in PR(s): #32167
  - File  RecoEgamma/EgammaElectronAlgos/src/GsfElectronAlgo.cc  modified in PR(s): #33666
  - File  RecoEgamma/EgammaElectronProducers/plugins/GsfElectronProducer.cc  modified in PR(s): #33666
  - File  RecoEgamma/EgammaPhotonProducers/src/GEDPhotonProducer.cc  modified in PR(s): #33666
  - File  RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py  modified in PR(s): #33548
  - File  RecoHGCal/TICL/plugins/TrackstersFromSimClustersProducer.cc  modified in PR(s): #33548
  - File  RecoHGCal/TICL/python/EMStep_cff.py  modified in PR(s): #33645
  - File  RecoHGCal/TICL/python/HADStep_cff.py  modified in PR(s): #33645
  - File  RecoHGCal/TICL/python/MIPStep_cff.py  modified in PR(s): #33645
  - File  RecoHGCal/TICL/python/SimTracksters_cff.py  modified in PR(s): #33548
  - File  RecoHGCal/TICL/python/TrkEMStep_cff.py  modified in PR(s): #33645
  - File  RecoHGCal/TICL/python/TrkStep_cff.py  modified in PR(s): #33645
  - File  RecoHGCal/TICL/python/iterativeTICL_cff.py  modified in PR(s): #33548, #33645
  - File  RecoPixelVertexing/PixelVertexFinding/plugins/gpuVertexFinderImpl.h  modified in PR(s): #33366
  - File  SimG4Core/Application/src/SteppingAction.cc  modified in PR(s): #33746
  - File  Validation/Configuration/python/hgcalSimValid_cff.py  modified in PR(s): #33548
  - File  Validation/HGCalValidation/plugins/HGCalHitValidation.cc  modified in PR(s): #33719
  - File  Validation/HGCalValidation/python/HGCalValidator_cfi.py  modified in PR(s): #33548
  - File  Validation/HGCalValidation/python/PostProcessorHGCAL_cfi.py  modified in PR(s): #33548
  - File  Validation/HGCalValidation/scripts/makeHGCalValidationPlots.py  modified in PR(s): #33548

cmsbuild commented:
Pull request #33612 was updated. @SiewYan, @andrius-k, @gouskos, @chayanit, @wajidalikhan, @ianna, @kpedro88, @ErnestaP, @Martin-Grunewald, @rekovic, @tlampen, @alberto-sanchez, @pohsun, @santocch, @cecilecaillol, @perrotta, @civanch, @yuanchao, @makortel, @srimanob, @ahmad3213, @cmsbuild, @agrohsje, @fwyzard, @GurpreetSinghChahal, @smorovic, @davidlange6, @smuzaffar, @Dr15Jones, @jpata, @cvuosalo, @mdhildreth, @jfernan2, @slava77, @ggovi, @qliphy, @fabiocos, @francescobrivio, @malbouis, @mkirsano, @jordan-martins, @kmaeshima, @emeschi, @silviodonato, @franzoni, @fgolf, @mariadalfonso, @rvenditti can you please check and sign again.
